argocd/overlays/moc-infra/applications/envs/moc/smaug/fybrik/arrow-flight-module.yaml

```diff
@@ -0,0 +1,8 @@
+project: fybrik
```

Comment:
you seem to be missing the metadata, apiversion, and spec fields (same with vault)

Reply:
@HumairAK I just added the required fields